libnixstore/shell.nix

```diff
@@ -0,0 +1,16 @@
+with import <nixpkgs> { };
```

Comment:
We dont really need another shell in that dir, could you remove it?

Reply:
So sorry about this, I did not intend to add that.